src/backend/joanie/core/models/accounts.py

```diff
@@ -15,3 +20,35 @@ class Meta:
 
     def __str__(self):
         return self.username
+
+
+class Address(models.Model):
+    """Address model stores address information (to generate bill after payment)"""
+
+    type = models.CharField(
```

Comment:
maybe we just need billing addresses at the moment

Reply:
+ 1